jsk_2016_01_baxter_apc/test/test-ik-in-bin.l

```diff
@@ -35,18 +44,9 @@
             (setq pos (float-vector tmp-x tmp-y tmp-z))
             (assert
               (or
-                (send robot :rarm :inverse-kinematics
-                  (make-cascoords :pos pos)
-                  :rotation-axis t
-                  :debug-view nil
-                  :warnp nil
-                  :dump-command nil)
-                (send robot :larm :inverse-kinematics
-                  (make-cascoords :pos pos)
-                  :rotation-axis t
-                  :debug-view nil
-                  :warnp nil
-                  :dump-command nil)
+                (ik-with-rotating-gripper robot :rarm 90 (make-cascoords :pos pos))
+                (ik-with-rotating-gripper robot :rarm 0 (make-cascoords :pos pos :rpy (float-vector 0 (deg2rad -90) 0)))
+                (ik-with-rotating-gripper robot :larm 90 (make-cascoords :pos pos))
```

Comment:
When `arm` in `(send robot :rotate-gripper arm gripper-angle :relative nil)` is `:larm`, nothing happens.